packages/core/src/Services/socket-general.js

```diff
@@ -159,13 +159,16 @@ const BinarySocketGeneral = (() => {
                     if (window.TrackJS) window.TrackJS.track('Custom InvalidToken error');
                 }
                 client_store.logout().then(() => {
+                    common_store.routeTo(routes.trade);
+                    /* keep this for future reference
                     common_store.setError(true, {
                         header: response.error.message,
                         message: localize('Please Log in'),
                         should_show_refresh: false,
                         redirect_label: localize('Log in'),
                         redirectOnClick: () => redirectToLogin(false, getLanguage()),
                     });
+                    */
```

Comment:
IMO we can just remove this comment. It is not like we want to bring it back anytime soon, and if we do want to bring this part back, we can just refer the commit.